### DISCUSSION TITLE
ClosestFeature "type" column

### DISCUSSION THREAD

**Discussion by dru93** [stuart-lab/signac#1448] [Q&A] [answered]
Hi, just a quick question: does the type column outputted from the ClosestFeature command refer to the transcript or the peak?
After running it I get four different types: cds, exon, gap and utr. Aren't utr and cds types part of exons, why are they considered as different types?

**Comment by timoast** [comment, marked as answer]
The `type` column originates from what is in the gene annotations. The `ClosestFeature` function will find the closest gene annotation feature to each input genomic ranges.

> Aren't utr and cds types part of exons, why are they considered as different types?

UTR and is different to an exon
